packages/system/kairos-agent/collection.yaml

```diff
@@ -1,7 +1,7 @@
 packages:
   - name: "kairos-agent"
     category: "system"
-    version: "2.13.7"
+    version: "2.14.0"
```

Comment:
there is no such version of kairos-agent (?)

Reply:
there is, but it should be an rc as I screwed up :( https://github.com/kairos-io/kairos-agent/releases/tag/v2.14.0